### ISSUE TITLE
allow import public keys up to 1Mb

### ISSUE THREAD

**Issue by DenBond7** [FlowCrypt/flowcrypt-android#906]
it relates to #902

**Comment by tomholub** [comment]
This is interesting - pgp keys can accumulate many 3rd party signatures on them. So many, that a key can have 20mb (if the key server accepted all of these signatures and is serving them to us).

The Node code would accept keys up to 1mb, and then strip all external signatures from it. Is something like that possible with PGPainless? Later we'll take a look.

**Comment by IvanPizhenko** [comment]
If I remember correctly, such thing was in the code that I've ported from TS, it was removing all non-self-signatures. But after porting that to PGPainless this functionality was lost. There seems to be no analog in PGPainless, so maybe it is easier just to take that part of code from previous version.

**Comment by tomholub** [comment]
When you say from previous version, do you mean to say we previously did it in Kotlin? Or do you mean to say we should port it from TypeScript into PGPainless?

**Comment by IvanPizhenko** [comment]
yes, by previous version I mean previous version in Kotlin. Yes, this can be made part of PGPainless, as yet another utility method.

**Comment by tomholub** [comment]
sounds good :+1:
